pilot/pkg/model/network_test.go

```diff
@@ -36,10 +36,10 @@ import (
 )
 
 func TestGatewayHostnames(t *testing.T) {
-	test.SetForTest(t, &model.MinGatewayTTL, 30*time.Millisecond)
+	test.SetForTest(t, &model.MinGatewayTTL, 110*time.Millisecond)
 
 	gwHost := "test.gw.istio.io"
-	workingDNSServer := newFakeDNSServer(":15353", 1, sets.New(gwHost))
+	workingDNSServer := newFakeDNSServer(":15353", 0, sets.New(gwHost))
```

Comment:
The idea is to test refresh rate of DNS records. Setting TTL to 0 would lose test coverage

Reply:
Probably need more context in the test to indicate that then, it's a non-explicit subtlety.

Reply:
@dddddai I was under the impression that, setting ttl to 0, we would have fallen back to `MinGatewayTTL` (in this case 110millis), so we would have been testing dns refresh rate with ttl 110millis.. are you saying that's not the case? can you clarify a bit? (I'm trying to understand if there is a way to reduce this test execution time, as if the minimum ttl possible is 1 second, then of course we can close this PR)

Reply:
Generally the refresh rate is the TTL returned by a DNS server. `MinGatewayTTL` is only used when the TTL is too low or the DNS server is unavailable.

In these tests we want to [check if the refresh rate is correct](https://github.com/istio/istio/pull/44384/files#diff-9935989f83680598f88529c13b6febf04af2267d8b89b90219e61eba147b9601R116) in different cases. Setting TTL to 0 means we will always use `MinGatewayTTL` as the refresh rate, so we would lose the coverage of cases where TTL is respected